src/app/user-navigation/user-navigation.component.ts

```diff
@@ -140,7 +140,7 @@ export class UserNavigationComponent implements OnInit {
     }
 
     ngOnInit(): void {
-        this.resetTheme('igx-indigo-light-theme');
+        // this.resetTheme('igx-indigo-light-theme');
         document.body.classList.add(this.theme.globalTheme);
```

Comment:
In this case we should add to classList directly the name of the initial theme, otherwise we are getting an error "globalTheme is not defined". We need to add the theme class like this:
document.body.classList.add('igx-indigo-light-theme');